### DISCUSSION TITLE
本机下载速度 上传速度 和 用户 下载速度 上传速度 是不是弄反了？

### DISCUSSION THREAD

**Discussion by SevendaysZzzz** [c0re100/qBittorrent-Enhanced-Edition#416] [General]
![image](https://user-images.githubusercontent.com/90546606/189512998-8fd1ebc7-12ba-4b5f-bfc0-362a6c357245.png)

有个疑惑，如上图。
我观察到，我本机下载 上传 速度 和 用户 下载 上传 速度 一样。
难道不应该是 本机下载速度 对应 用户上传速度，本机上传速度 对应 用户下载速度吗？
这里是不是弄反了?

**Comment by andry81** [comment]
Which one in the image download and upload speed?

<details>
<summary>Next time translate it</summary>
<a href="https://translate.yandex.com/ocr">https://translate.yandex.com/ocr</a><br />
<img src="https://user-images.githubusercontent.com/16258197/189531986-feaa1faf-2841-4df0-a9e9-19704af6c47e.png" />
</details>

**Comment by SevendaysZzzz** [comment]
I mean my download speed should be equal to the sum of upload speed of all peer users and my upload speed should be equal to the sum of download speed of all peer users. However, the reality is the opposite.

**Comment by avasummer** [comment]
这个用户会不会有可能是你自己？

**Comment by wesley800** [comment]
没反，这里就是指你从该用户下载和上传的对应速度。
